Decisions.md

```diff
@@ -0,0 +1,29 @@
+# Decisions
+
+## Voting
+
+Decisions are based on voting using the following:
+
+* +1 : For
+* +0: Mildly for, but mostly indifferent
+* 0: Indifferent
+* -0: Mildly against, but mostly indifferent
+* -1 : Strongly against. This is expected to be accompanied by a reason.
+
+For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. 
+
```

Comment:
```suggestion

The result is determined by a simple majority (50%+1 of the total votes cast).
In the rare case where the larger community votes in a way contrary to the PSC, the decision is considered contentious.
```

Reply:
This encodes the conversation below, but raises an interesting point. Consider this hypothetical:

* The PSC decides they want to have Terrible McTerrible lead the project as Dictator-in-chief
* The community (correctly) think this a bad idea
* The PSC all vote +1 because Terrible McTerrible has promised to make them all rich
* The community all vote -1
* Decision becomes contentious, and goes to PSC to decide
* PSC vote for McTerrible
* Hilarity ensues

Obviously, it assumes that the PSC could all be suckered in by some bad actor, and I'm sure none of you would be that gullible, but I think you might want to consider a "we have a terrible PSC" clause somewhere, perhaps. Of course, on the flipside, it doesn't rule out branch stacking :woman_shrugging: